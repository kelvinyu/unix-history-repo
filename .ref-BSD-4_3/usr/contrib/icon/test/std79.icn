record array(a,b,c,d,e,f,g)

procedure dummy(u,v,x,y,z)
   suspend u | v
   return x
end

procedure main()
   write("x := 1 ----> ",image(x := 1) | "none")
   write("y := 2 ----> ",image(y := 2) | "none")
   write("x <-> y ----> ",image(x <-> y) | "none")
   write("y <-> x ----> ",image(y <-> x) | "none")
   write("(x <-> y) & &fail ----> ",image((x <-> y) & &fail) | "none")
   write("x ----> ",image(x) | "none")
   write("y ----> ",image(y) | "none")
   write("(1 <-> y) & &fail ----> ",image((1 <-> y) & &fail) | "none")
end
