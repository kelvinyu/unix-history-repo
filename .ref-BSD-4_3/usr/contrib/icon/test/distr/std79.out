x := 1 ----> 1
y := 2 ----> 2
x <-> y ----> 2
y <-> x ----> 2
(x <-> y) & &fail ----> none
x ----> 1
y ----> 2
Run-time error 111 at line 16 in std79.icn
variable expected
offending value: 1
